### PR TITLE
Added Accessing hosts topic

### PR DESCRIPTION
**Pull request by ahardin-rh**
Per https://bugzilla.redhat.com/show_bug.cgi?id=1753034

Preview build: http://file.rdu.redhat.com/~ahardin/10242019/accessing-hosts-on-aws/networking/accessing-hosts.html

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@sferich888 @wking Can you please take a look at my first pass at adding these docs in? Thanks!

**Comment by sferich888** [comment]
This is specific to IPI installs; UPI installs on AWS likely have to use similar information, and that needs to be denoted somewhere. 

We also need to include similar info for the other platforms we'll be supporting.

File: modules/accessing-hosts-on-aws.adoc [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+. Create a security group that allows SSH access into the virtual private cloud
+(VPC) created by the `openshift-install` command.
+
+. Create an Amazon EC2 instance inside the VPC that the installer created or on
+one of the public subnets the installer created.
+
+. Associate a public IP address with the Amazon EC2 instance that you created.
++
+Unlike with the {product-title} installation, you must associate the Amazon EC2
+instance you created with an SSH keypair. It does not matter what operating
```
Comment (wking) [inline modules/accessing-hosts-on-aws.adoc]: Depends on which AMI you use.  With RHCOS, you can provide keys via Ignition, like the installer does.  So we should at least soften the "must" or more clearly scope it to a particular AMI family.

File: modules/accessing-hosts-on-aws.adoc [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+system you choose for this instance, as it will simply serve as an SSH bastion
+to bridge the internet into your {product-title} cluster's VPC.
+
+. Once you provisioned your Amazon EC2 instance and can SSH into it, you must add
+the SSH key that you associated with your {product-title} installation, not the
+key for the bastion instance.
```
Comment (wking) [inline modules/accessing-hosts-on-aws.adoc]: These could be different keys, but tgey don't have to be.

File: modules/accessing-hosts-on-aws.adoc [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+. Run `oc get nodes`, inspect the output, and choose one of the nodes that is a
```
Comment (wking) [inline modules/accessing-hosts-on-aws.adoc]: Somewhere we want to say that direct SSH access is only recommended for disaster recover.  When the Kubernetes API is responsive, as you're assuming here, folks should run (privileged) pods instead.

**Comment by ahardin-rh** [comment]
@sferich888 @wking Thanks for your feedback! I made some updates. PTAL.

Preview build: http://file.rdu.redhat.com/~ahardin/10242019/accessing-hosts-on-aws/networking/accessing-hosts.html

**Comment by ahardin-rh** [comment]
@sferich888 @wking Can you please review the latest changes? http://file.rdu.redhat.com/~ahardin/10242019/accessing-hosts-on-aws/networking/accessing-hosts.html Thanks!

**Review by sheriff-rh — APPROVED** [review]
Looks great, Ashley!

**Comment by ahardin-rh** [comment]
@rbbratta Can you please provide a QE review or point me to the correct contact who can help? Thanks!

**Comment by ahardin-rh** [comment]
@lihongan Are you the correct contact to provide QE review? Thank you!

File: modules/accessing-hosts-on-aws.adoc (hunk 3 above)
Comment (ahardin-rh) [reply modules/accessing-hosts-on-aws.adoc]: Thanks! I added a note

**Comment by ahardin-rh** [comment]
@xltian Can you please help me track down a correct QE contact for this? Thank you!

**Comment by xltian** [comment]
@jianlinliu  will take a look from QE.

File: modules/accessing-hosts-on-aws.adoc [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+the Amazon Elastic Compute Cloud (Amazon EC2) instances that it provisions for
+your {product-title} cluster. In order to be able to SSH to your {product-title}
+hosts, you must follow this procedure.
+
+.Procedure
+
+. Create a security group that allows SSH access into the virtual private cloud
+(VPC) created by the `openshift-install` command.
+
+. Create an Amazon EC2 instance inside the VPC that the installer created or on
```
Comment (jianlinliu) [inline modules/accessing-hosts-on-aws.adoc]: If want to user can ssh login bastion host, I think `Create an Amazon EC2 instance on one of the public subnets the installer created` will be more accurate, no need other option, such as: `inside the VPC that the installer created`.

Comment (ahardin-rh) [reply modules/accessing-hosts-on-aws.adoc]: Thank you!

**Pull request merged**